.gitignore

```diff
@@ -25,4 +25,6 @@ supabase/.branches
 supabase/.temp
 
 dist/
-.env
\ No newline at end of file
+.env
+
+deno.lock
```

Comment:
Not sure if we should commit this file or not... :thinking: 
It is auto-generated, but couldn't find for sure if I'm supposed to version it or not.
Do you guys know?

Reply:
If it's a lock file, it should probably be commited

Reply:
committed